docs/install/index.rst

```diff
@@ -3,7 +3,7 @@
     distributed with this work for additional information
     regarding copyright ownership.  The ASF licenses this file
     to you under the Apache License, Version 2.0 (the
-    "License"); you may not use this file except in compliance
+  d  "License"); you may not use this file except in compliance
```

Comment:
It appears that this license notice is included across tons of .rst files. Could you elaborate on the details of this change?